src/test/java/amazon/review/ReviewApplicationTests.java

```diff
@@ -1,13 +1,40 @@
 package amazon.review;
 
+import amazon.review.service.CsvReaderService;
+import java.util.List;
+import org.junit.Assert;
 import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.context.SpringBootTest;
+import static org.junit.jupiter.api.Assertions.assertThrows;
 
 @SpringBootTest
 class ReviewApplicationTests {
```

Comment:
You might want to move the tests into a separate file, like `CsvReaderServiceImplTests`

Reply:
oh, ok, thanks!